frontend/src/metabase/nav/components/search/RecentsList/util.ts

```diff
@@ -14,5 +14,7 @@ export const isItemActive = ({ model, model_object }: RecentItem) => {
   return isSyncCompleted(model_object);
 };
 
-export const getItemUrl = (item: RecentItem) =>
-  isItemActive(item) ? Urls.modelToUrl(item) : "";
+export const getItemUrl = (item: RecentItem) => {
+  const url = isItemActive(item) && Urls.modelToUrl(item);
```

Comment:
- Ensures that we only get a string or `undefined`
- Uses `undefined` because `href`  on `Anchor` only accepts `string | undefined` so just adapting to that.